docker-compose.yml

```diff
@@ -4,4 +4,10 @@ services:
     image: php:7.3
     volumes:
      - .:/opt/project
-    command: /opt/project/vendor/bin/phpunit /opt/project/tests/TestStubs.php
```

Comment:
Because of this error from travis-ci... but I only fixed the travis build, maybe I missed something else? 
```
docker-compose -f docker-compose.yml run php
Creating network "phpstormstubs_default" with the default driver
/usr/local/bin/docker-php-entrypoint: 9: exec: /opt/project/vendor/bin/phpunit: not found
The command "docker-compose -f docker-compose.yml run php" exited with 127.
```

Reply:
That was my fault. I fixed that with https://github.com/JetBrains/phpstorm-stubs/commit/e5f2b11c4d8037bd6e66107fbf590334375d18ea this commit today.